### ISSUE TITLE
NTR: Leishmania in bone marrow, spleen, liver, or skin biopsies (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9427]
New term request Leishmania in bone marrow, spleen, liver, or skin biopsies (nadea3_190502021617): Biopsy; Wright or Giemsa stain

**Comment by pnrobinson** [comment]
@MickeySegal

There is a related term [Tegumentary leishmaniasis susceptibility HP:0007408](https://hpo.jax.org/app/browse/term/HP:0007408)

If this is not sufficient then we should probably make terms that are specific. I think the most common affected organs are skin, spleen, liver, and bone marrow

**Comment by MickeySegal** [comment]
This is not the susceptibility; this is finding the parasites.  I'd be fine with either a generic term for finding the parasites in various places or separate terms for finding them in these 4 places.

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000255
- label: Demonstration of Leishmania in tissue
- parents: [Positive infectious agent test (HP:0430030)](https://hpo.jax.org/app/browse/term/HP:0430030)
- pmid: 
- def: Presence of Leishmania, a parasitic protozoan, in a tissue specimen. Leishmania is responsible for leishmaniasis.
- comment: Leishmania may be detected in tissues such as bone marrow, spleen, liver, or skin by methods including Wright or Giemsa stain.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9427
- synonyms:
